misk-aws/src/main/kotlin/misk/jobqueue/sqs/QueueResolver.kt

```diff
@@ -95,7 +91,6 @@ val QueueName.retryQueue
 
 val QueueName.parentQueue
   get() = when {
-    isDeadLetterQueue -> QueueName(value.removeSuffix(deadLetterQueueSuffix))
```

Comment:
Would we still want to keep this in if the queue is using its own DLQ?

Reply:
Good catch! I thought I reverted that with the (force) push of 9f9ccf9 but looks like I didn't  properly amend that commit.

I believe this particular `when` condition can no longer be met after this change but I didn't mean to remove it; will restore before merging.